src/OrasProject.Oras/Utils/MediaType.cs

```diff
@@ -11,23 +11,11 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-namespace OrasProject.Oras;
+namespace OrasProject.Oras.Utils;
```

Comment:
Shouldn't it just be 
```suggestion
namespace OrasProject.Oras;
```
Avoid using any `Utils` namespaces in a library. Anything in the `Utils` should be placed in a right place.